inc/saitunnel.h

```diff
@@ -914,13 +914,13 @@ typedef enum _sai_tunnel_term_table_entry_type_t
     /** Tunnel termination table point to point entry match on dst & src IP & tunnel type */
     SAI_TUNNEL_TERM_TABLE_ENTRY_TYPE_P2P,
 
-    /** Tunnel termination table point to multi point entry match on src IP & dst IP+mask & tunnel type */
+    /** Tunnel termination table point to multi point entry match on dst IP & src IP+mask & tunnel type */
     SAI_TUNNEL_TERM_TABLE_ENTRY_TYPE_P2MP,
 
-    /** Tunnel termination table multi point to point entry match on src IP+mask & dst IP & tunnel type */
+    /** Tunnel termination table multi point to point entry match on dst IP+mask & src IP & tunnel type */
```

Comment:
MP2P = (multiple destinations from one source)
Please add this comment inline here